### DISCUSSION TITLE
400 Error when selecting a table with an eq filter by the ID (uuid)

### DISCUSSION THREAD

**Discussion by RafiulM** [supabase/supabase#12002] [Questions]
I'm trying to fetch data from supabase with the following code :

`async function getData() {
    const { data, error } = await supabase
      .from('customer')
      .select('*')
      .eq('id', '50f18eee-a6dc-4075-9b82-ab4e02856786')
    console.log(data)
  }`

but it's returning this error :
`GET https://nptaexhcclayodlqmtdx.supabase.co/rest/v1/customer?select=*&id=eq.50f18eee-a6dc-4075-9b82-ab4e02856786 400`

I'm not sure what's causing the error, since the value does exist in the "id" column. This is the response I get when querying select('id') :

`0
{id: '50f18eee-a6dc-4075-9b82-ab4e02856786'}`

**Comment by GaryAustin1** [comment]
Are you getting any error info in either error or the response?
Even if it does not find a match for the filter or is blocked by RLS it should return empty array and not error.

**Reply by RafiulM** [reply]
when I console.log data and response, neither returns any value
